splinterd/man/splinterd.1.md

```diff
@@ -324,6 +324,12 @@ Cylinder JWT, and OAuth.
 
 Cylinder JWT authorization is enabled by default.
 
+A Cylinder JWT is a custom JSON Web Token implementation which supports the
+signing algorithms provided the by the Cylinder library.  This includes
```

Comment:
s/the by the/by the/

Reply:
According to the doc guidelines, should only have a single space between sentences.

Reply:
Fixed